fluent-syntax/src/parser.js

```diff
@@ -212,7 +212,7 @@ export default class FluentParser {
       tags = this.getTags(ps);
     }
 
-    if (pattern === undefined && attrs === undefined && tags === undefined) {
```

Comment:
My understanding was that we don't accept messages with just tags (they have to have attrs or value). Am I wrong?